dnn/variational_inference.ipynb

```diff
@@ -7,7 +7,7 @@
         "colab_type": "text"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def valid_in_all_rasters(lat: float, lon: float, rasters: List[raster.AmazonGeoTiff]) -&gt; bool:
</pre><p>should collapse the data prep code block by default</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>